CMakeLists.txt

```diff
@@ -748,15 +745,51 @@ if(BUSTED_LUA_PRG)
     set_target_properties(functionaltest-lua PROPERTIES FOLDER test)
 endif()
 
-if(LUACHECK_PRG)
-  add_custom_target(lualint
-    COMMAND ${LUACHECK_PRG} -q runtime/ scripts/ src/ test/
-    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
-else()
-  add_custom_target(lualint false
-    COMMENT "lualint: LUACHECK_PRG not defined")
-endif()
-
+find_program(LUACHECK luacheck)
```

Comment:
the `_PRG` suffix convention was intentional and used throughout our cmake files. Why revert that?

Reply:
Found it too wordy, don't mind either way.

Reply:
`_PRG` convention should be kept